book/pose.html

```diff
@@ -1520,10 +1523,10 @@ <h1><a href="index.html" style="text-decoration:none;">Robotic Manipulation</a><
           scene. 
           <ol>
             <li>First, write down each of the correspondence, i.e the ($b_{i},
-            r_{j}$) pairs. Next, compute the ICP error, the sum of the pairwise
+            r_{j}$) pairs. Next, compute the ICP error, the sum of the pairwise squared
             distances between each of these correspondences. </li>
             <li>Suppose that you are told that $\frac{1}{3}$ of the scene
-            points are outliers. Using the pairwise distances between each
+            points are outliers. Using the pairwise squared distances between each
```

Comment:
Don't think we need to make the change here, since it doesn't matter

Reply:
Removed!